onnx/onnx-data.in.proto

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) ONNX Project Contributors.
+// Licensed under the MIT license.
```

Comment:
what's the difference between onnx-data.proto and onnx-data-ml.proto?

Reply:
No difference. It was automatically generated when ONNX_ML=1 using gen_proto.py.

Reply:
that's bad. if no difference needed, then no need having two files. "// #if ONNX-ML" should not in the file.